tendrl/gluster_integration/manager/manager.py

```diff
@@ -200,6 +266,27 @@ def on_pull(self, raw_data, cluster_id):
                     index += 1
                 except KeyError:
                     break
```

Comment:
As discussed, we need to create a well formed output file so that we can handle the error conditions appropriately. Can do it as a separate patch, please raise an issue and attach here